### PR TITLE
[hotfix][docs]Review to reduce passive voice, improve grammar and formatting

### PR DESCRIPTION
**Pull request by ChrisChinchilla**
Small review of the runtime concept doc to reduce passive voice, reduce future tense, improve grammar and formatting. Let me know if it needs backporting to any other branch or if there are any other issues

### CODE REVIEW

File: docs/concepts/runtime.md [hunk 1]
```diff
@@ -88,40 +94,36 @@ By default, Flink allows subtasks to share slots even if they are subtasks of di
...
+[Savepoints](../ops/state/savepoints.html) are **manually triggered checkpoints**, which take a snapshot of the program and write it out to a state backend. They rely on the regular checkpointing mechanism for this. During execution, programs are periodically snapshotted on the worker nodes and produce checkpoints. You only need the last completed checkpoint for recovery, and you can safely discard older checkpoints as soon as a new one is completed.
 
-Savepoints are similar to these periodic checkpoints except that they are **triggered by the user** and **don't automatically expire** when newer checkpoints are completed. Savepoints can be created from the [command line](../ops/cli.html#savepoints) or when cancelling a job via the [REST API](../monitoring/rest_api.html#cancel-job-with-savepoint).
+Savepoints are similar to these periodic checkpoints except that they are **triggered by the user** and **don't automatically expire** when newer checkpoints are completed. You can create savepoints can from the [command line](../ops/cli.html#savepoints) or when canceling a job via the [REST API](../monitoring/rest_api.html#cancel-job-with-savepoint).
```
Comment (greghogan) [inline docs/concepts/runtime.md]: "savepoints can" -> "savepoints"

File: docs/concepts/runtime.md [hunk 2]
```diff
@@ -28,12 +28,12 @@ under the License.
 
 ## Tasks and Operator Chains
 
-For distributed execution, Flink *chains* operator subtasks together into *tasks*. Each task is executed by one thread.
+For distributed execution, Flink *chains* operator subtasks together into *tasks*, with one thread executing each task.
 Chaining operators together into tasks is a useful optimization: it reduces the overhead of thread-to-thread
-handover and buffering, and increases overall throughput while decreasing latency.
-The chaining behavior can be configured; see the [chaining docs](../dev/datastream_api.html#task-chaining-and-resource-groups) for details.
+handover and buffering and increases overall throughput while decreasing latency.
+You can configure the chaining behavior, read the [chaining docs](../dev/datastream_api.html#task-chaining-and-resource-groups) for details.
```
Comment (greghogan) [inline docs/concepts/runtime.md]: Preserve the semi-colon?

File: docs/concepts/runtime.md [hunk 3]
```diff
@@ -28,12 +28,12 @@ under the License.
...
 
-The sample dataflow in the figure below is executed with five subtasks, and hence with five parallel threads.
+Five subtasks execute the sample data flow in the figure below with five parallel threads.
```
Comment (greghogan) [inline docs/concepts/runtime.md]: The community as well as other projects refer to "dataflow" extensively (also below).

File: docs/concepts/runtime.md [hunk 4]
```diff
@@ -46,19 +46,23 @@ The Flink runtime consists of two types of processes:
   - The **JobManagers** (also called *masters*) coordinate the distributed execution. They schedule tasks, coordinate
     checkpoints, coordinate recovery on failures, etc.
 
-    There is always at least one Job Manager. A high-availability setup will have multiple JobManagers, one of
-    which one is always the *leader*, and the others are *standby*.
+    There is always at least one Job Manager. A high-availability setup should have multiple JobManagers, one of
```
Comment (greghogan) [inline docs/concepts/runtime.md]: "JobManager"?

File: docs/concepts/runtime.md [hunk 5]
```diff
@@ -88,40 +94,36 @@ By default, Flink allows subtasks to share slots even if they are subtasks of di
...
+    slotted resources, while making sure that the heavy subtasks are evenly distributed among the TaskManagers.
 
 <img src="../fig/slot_sharing.svg" alt="TaskManagers with shared Task Slots" class="offset" width="80%" />
 
-The APIs also include a *[resource group](../dev/datastream_api.html#task-chaining-and-resource-groups)* mechanism which can be used to prevent undesirable slot sharing. 
+The APIs also include a *[resource group](../dev/datastream_api.html#task-chaining-and-resource-groups)* mechanism which you can use to prevent undesirable slot sharing.
 
-As a rule-of-thumb, a good default number of task slots would be the number of CPU cores.
-With hyper-threading, each slot then takes 2 or more hardware thread contexts.
+As a rule-of-thumb, a reasonable default number of task slots would be the number of CPU cores. With hyper-threading, each slot then takes 2 or more hardware thread contexts.
```
Comment (greghogan) [inline docs/concepts/runtime.md]: Preserve the line split? I'm not sure what the second sentence is saying. Typically hyper-threads are reported as separate cores so would not each slot take a single hardware thread context?

File: docs/concepts/runtime.md [hunk 6]
```diff
@@ -88,40 +94,36 @@ By default, Flink allows subtasks to share slots even if they are subtasks of di
...
 
 {% top %}
 
 ## State Backends
 
-The exact data structures in which the key/values indexes are stored depends on the chosen [state backend](../ops/state/state_backends.html). One state backend
-stores data in an in-memory hash map, another state backend uses [RocksDB](http://rocksdb.org) as the key/value store.
-In addition to defining the data structure that holds the state, the state backends also implement the logic to
-take a point-in-time snapshot of the key/value state and store that snapshot as part of a checkpoint.
+The exact data structures which store the key/values indexes depends on the chosen [state backend](../ops/state/state_backends.html). One state backend stores data in an in-memory hash map, another state backend uses [RocksDB](http://rocksdb.org) as the key/value store. In addition to defining the data structure that holds the state, the state backends also implement the logic to take a point-in-time snapshot of the key/value state and store that snapshot as part of a checkpoint.
```
Comment (greghogan) [inline docs/concepts/runtime.md]: Preserve the line splits? The conversion to HTML ignores single newlines.

File: docs/concepts/runtime.md (hunk 4 above)
Comment (alpinegizmo) [reply docs/concepts/runtime.md]: "A high-availability setup should have multiple JobManagers" is, in general, not true -- this is a detail that depends on the underlying cluster management framework. 

I suggest reworking as follows:

There is always at least one Job Manager, but some [high-availability setups]({{ site.baseurl }}/ops/jobmanager_high_availability.html) will have multiple JobManagers.

**Comment by aljoscha** [comment]
I'm closing this as "Abandoned", since there is no more activity and the code base has moved on quite a bit. Please re-open this if you feel otherwise and work should continue.

**Pull request closed without merging**